src/CanvasInput/index.js

```diff
@@ -59,47 +53,40 @@ export default class CanvasInput extends React.Component {
     }
   }
 
-  handleClick = evt => {
-    evt.stopPropagation()
-    evt.preventDefault()
-    if (!this.state.mouseDrag) {
-      const mouse = getMouseFromEvent(evt)
-      const solvedData = runHoverSolverOn(
-        getDataUnderMouse(this.props, mouse, this.canvasRef.current)
-      )
-      this.props.onUpdate({
-        action: 'mouseClick',
-        mouse,
-        renderDatum: solvedData.renderDatum,
-        hoverRenderData: solvedData.hoverRenderData,
-        hoverData: solvedData.hoverData,
-        localMouse: solvedData.localMouse,
-        layerProps: solvedData.layerProps,
-        rootProps: this.props.rootProps,
-      })
-    }
-    this.setState({mouseDrag: false})
+  handleClick = event => {
+    event.stopPropagation()
+    event.preventDefault()
+    event.persist()
```

Comment:
Is this so that you can access `event` inside of `setState`? If so, it might be easier to move `const mouse = getMouseFromEvent(event)` outside.

Reply:
It was, but it's not needed anymore as per your comment below